### PR TITLE
Added managing roles on development branch

### PR DESCRIPTION
**Pull request by Edafan4uk**

### CODE REVIEW

**Review by NazarStasiv — APPROVED** [review]

File: TravelingBlog.Angular/package.json [hunk 1]
```diff
@@ -20,6 +20,8 @@
     "core-js": "2.4.1",
     "jquery": "3.3.1",
     "jqueryui": "1.11.1",
+    "jwt-decode": "^2.2.0",
```
Comment (Alexel37) [inline TravelingBlog.Angular/package.json]: Use stable version instead of latest!

File: TravelingBlog.Angular/package.json [hunk 2]
```diff
@@ -20,6 +20,8 @@
...
+    "ngx-bootstrap": "^3.1.4",
```
Comment (Alexel37) [inline TravelingBlog.Angular/package.json]: Use stable version instead of latest!

File: TravelingBlog.Angular/package.json (hunk 1 above)
Comment (Edafan4uk) [reply TravelingBlog.Angular/package.json]: ok
```suggestion
    "jwt-decode": "2.2.0",
```

File: TravelingBlog.Angular/package.json (hunk 2 above)
Comment (Edafan4uk) [reply TravelingBlog.Angular/package.json]: Is that good?
```suggestion
    "ngx-bootstrap": "3.1.4",
```

**Review by IraVolkoshevych — APPROVED** [review]

File: TravelingBlog.Angular/package.json (hunk 1 above)
Comment (Edafan4uk) [reply TravelingBlog.Angular/package.json]: Resolved conflicts and fixed

**Review by DanyloMonets — APPROVED** [review]

**Pull request merged**